internal/api.go

```diff
@@ -156,6 +156,12 @@ func (c *APIClient) Post(url, contentType string, body io.ReadSeeker) (resp *htt
 	// transfer encoding.
 	if rb, ok := body.(*RingBuffer); ok {
 		req.ContentLength = int64(rb.Len())
+		// Fix the content-type header to avoid chunked transfer encoding
+		// which is not supported by healthchecks.io.
+		// Solve pings with empty bodies--i.e. command with no stdout/err
```

Comment:
Keep the comments brief. Id est in here is not necessary.
Maybe just say "Handle empty body"